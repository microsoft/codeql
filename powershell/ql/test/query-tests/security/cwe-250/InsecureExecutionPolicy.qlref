queries/security/cwe-250/InsecureExecutionPolicy.ql