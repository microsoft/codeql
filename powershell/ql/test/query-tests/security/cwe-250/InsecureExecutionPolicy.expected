| test.ps1:1:1:1:33 | Call to set-executionpolicy | Insecure use of 'Set-ExecutionPolicy'. |
| test.ps1:5:1:5:54 | Call to set-executionpolicy | Insecure use of 'Set-ExecutionPolicy'. |
| test.ps1:7:1:7:39 | Call to set-executionpolicy | Insecure use of 'Set-ExecutionPolicy'. |
